src/paho/mqtt/client.py

```diff
@@ -1762,6 +1771,8 @@ def loop_misc(self) -> MQTTErrorCode:
             if self._state == mqtt_cs_disconnecting:
                 rc = MQTTErrorCode.MQTT_ERR_SUCCESS
             else:
+                self._state = ConnectionState.MQTT_CS_CONNECTION_LOST
+                self._easy_log(MQTT_LOG_DEBUG, "... 2")
```

Comment:
```suggestion
```

Reply:
My bad, I should re-read my own PR more carefully :(